crates/turbopack-ecmascript/js/src/runtime.js

```diff
@@ -1221,8 +1251,12 @@ function disposeChunk(chunkPath) {
     if (noRemainingChunks) {
       moduleChunksMap.delete(moduleId);
       disposeModule(moduleId, "clear");
+      disposedModules.push(moduleId);
     }
   }
+  for (const moduleId of disposedModules) {
+    delete moduleCache[moduleId];
+  }
```

Comment:
```suggestion
```

No longer necessary as it's done in `disposeModule` (can remove `disposedModules`)